build.gradle

```diff
@@ -339,8 +359,14 @@ task prepareBwcTests {
 task "${baseName}#mixedClusterTask"(type: StandaloneRestIntegTestTask) {
     useCluster testClusters."${baseName}0"
     dependsOn "${baseName}#oldVersionClusterTask0"
-    doFirst {
-        testClusters."${baseName}0".upgradeNodeAndPluginToNextVersion(plugins)
+    if (bwcBundleTest){
+        doFirst {
+            testClusters."${baseName}0".nextNodeToNextVersion()
```

Comment:
How is the plugin getting upgraded here if we aren't calling `upgradeNodeAndPluginToNextVersion` ? https://github.com/opensearch-project/OpenSearch/blob/6a5b464c71964800507d389cfb0ae2d358175857/buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchCluster.java#L451

Reply:
oh i see its going to use the bundled version of the plugin.  If this is the case how are we testing local changes here against the bundle? That would defeat the purpose of having this as a pr check...

Reply:
Local changes are not tested. Made a note of this in the readme and removed bundle level tests from repo ci workflow.